vcl_templates/www.vcl.erb

```diff
@@ -296,6 +296,9 @@ sub vcl_deliver {
     add resp.http.Set-Cookie = "ABTest-Example=" req.http.GOVUK-ABTest-Example "; expires=" now + 1d;
   }
 
+  # Set the tls version cookie with the raw protocol version from fastly
+  add resp.http.Set-Cookie = "TLSversion=" tls.client.protocol "; max-age=0; secure";
```

Comment:
I might have just misunderstood this, but wouldn't `max-age=0` mean that the browse never sets the cookie? I thought this tended to be used to delete existing cookies.